san/available_metrics.py

```diff
@@ -1,4 +1,12 @@
+import inspect
 import san.sanbase_graphql
 
 def available_metrics():
-    return san.sanbase_graphql.available_metrics()
+    all_functions = inspect.getmembers(san.sanbase_graphql, inspect.isfunction)
+    all_functions = list(map(
+        lambda x: x[0] if x[0][0] != '_' else all_functions.remove(x), all_functions
```

Comment:
What is this?

Reply:
Also, to every element, you're mapping either the whole list or the whole list with 1 element removed?
I guess it's holding a reference and removing something at iteration N affects the element at position N - 1? 
Either way, please rewrite this

Reply:
You're right, should be okay now